scripts/publish/publish.ts

```diff
@@ -99,12 +104,20 @@ async function switchToNewBranch(branchName: string) {
 }
 
 async function bumpVersion(version) {
-    await run(`npm version --no-git-tag-version --allow-same-version ${version}`);
-    success(`Updated version in package.json to ${version}`);
+    const currentBranch = getCurrentBranch();
+    const nextVersion = calcNextVersion(version);
+    const bumpBranch = `release/bump-${nextVersion}`;
+    await run(`git checkout master`);
+    await run(`git checkout -b ${bumpBranch}`);
+    await run(`npm version --no-git-tag-version --allow-same-version ${nextVersion}`);
+
+    await run(`git commit -am "Bump version to ${nextVersion}"`);
+    await run(`get checkout "${currentBranch}"`);
```

Comment:
Should we be pushing the version bump branch here? What happens if the version bump branch already exists?

Also, looks like this should be `git checkout -b ${currentBranch}`

Reply:
Um, it doesn't have to have `-b` because the branch already exists. But the command is definitely `git` and not `get`. Whoops.

And yes, we should be pushing to remote. I'm adding a `--force-with-lease` in case the branch already exists.

Reply:
Changes look good to me